CleanArchitectureRxSwift/Scenes/AllPosts/PostItemViewModel.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  PostItemViewModel.swift
+//  CleanArchitectureRxSwift
+//
+//  Created by Stefano Mondino on 09/08/17.
+//  Copyright © 2017 sergdort. All rights reserved.
+//
+
+import Foundation
+import Domain
+
+final class PostItemViewModel   {
+    var title:String
+    var subtitle : String
+    var post: Post
```

Comment:
I agree, they should be `let`. 
Regarding the `struct` vs `class`, I disagree.
If you're building a whole architecture with view models, you would end up using some associated objects (from objc runtime). More than this, a view model has a mutable nature, since its properties can be updated by the view (with or without Rx). While in this specific case, `struct` may seem more appropriate, it shouldn't be a default in a complex application.